dev/com.ibm.ws.concurrent.persistent_fat_demo_timers/test-application/demotimers/src/ejb/timers/AutomaticDatabase.java

```diff
@@ -33,7 +33,7 @@
  * This class uses the @Schedule annotation.
  * Using this annotation will start the timer immediately on start and will run every 30 seconds.
  */
-@Stateless
+@Singleton //Ensure only one instance of this Timer is ever created.
```

Comment:
Singleton does seem convenient for restricting the table setup, but I think there may be other consequences for timers with respect to locking, so we need to ensure that is really what we want.